src/ansiblelint/rules/YamllintRule.py

```diff
@@ -79,24 +46,23 @@ def matchyaml(self, file: Lintable) -> List["MatchError"]:
         if str(file.base_kind) != "text/yaml":
             return matches
 
-        if YamllintRule.config:
```

Comment:
I dropped this conditional check as well since yamllint loading is no longer conditional.